include/openmc/random_ray/vtk_plot.h

```diff
@@ -0,0 +1,28 @@
+#ifndef OPENMC_RANDOM_RAY_VTK_PLOT_H
+#define OPENMC_RANDOM_RAY_VTK_PLOT_H
+
+#include "openmc/random_ray/random_ray.h"
+#include "openmc/random_ray/source_region.h"
+
+namespace openmc {
+void plot_3D_vtk();
+
+// Returns the inputted value with its
+// endianness reversed. This is useful
+// for conforming to the paraview VTK
+// binary file format.
+template<typename T>
```

Comment:
Do we actually need to flip endianness? I was under the impression that this is inferred under the hood by VTK when reading.

https://stackoverflow.com/questions/19377028/are-vtk-files-endian-independent-when-read-in-visualization-softwares-such-as-pa